### DISCUSSION TITLE
Content on Hover or Focus (Level AA)

### DISCUSSION THREAD

**Discussion by cporter-valassis** [mantinedev/mantine#5628] [Feature requests]
WCAG (level AA) is needed for components within Mantine that use "Tooltip." Please see the link for additional guidance below. 

https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

**Comment by rtivital** [comment]
You can change events that trigger tooltip – https://mantine.dev/core/tooltip/#change-events
